### DISCUSSION TITLE
Chia Directory WARNING / over 1,000 plots and havent won yet.

### DISCUSSION THREAD

**Discussion by simsim201** [Chia-Network/chia-blockchain#10345] [Support]
I have over a 1000 plots. its fully synced but I haven't WON yet. does anyone know what is going on? 

When I open the log file I am getting these two issues below and I was wondering if someone can Please help me. 

**1. : WARNING  Directory: C:\Users\MyComp\AppData\Local\chia-blockchain\app-1.2.10\-b3390 does not exist.**

**2. 2022-02-20T20:32:07.534 full_node chia.rpc.rpc_server     : WARNING  Sending data failed. Exception Traceback (most recent call last):**

I have the Latest Chia software 1.2.11
I also installed the older chia and I see the same issue. 

Please help
Thanks

**Comment by Kycin1** [comment]
Well like you said you only have 000 plots. I have been farming since april 2021 I have won zero as well. its just shit luck.

**Comment by Kycin1** [comment]
HAHA serious note I Don't know if those warnings are an issue sorry. I Had two computers running full nodes for 6 months before I finally read it was a nono. So If I was you I would just go through and start from the beginning. Check your port make sure each computer you have is set correctly to not interfere with your main node that is checking your proofs etc.  and it could all be A OK and like I said shit luck.
